pip_lock.py

```diff
@@ -4,9 +4,13 @@
 import os
 import sys
 
-from pip.operations.freeze import freeze as pip_freeze
+# pip 10.0.0 made freezer internal. Try to use new name and fallback to old one.
+try:
+    from pip._internal.operations.freeze import freeze as pip_freeze
+except ImportError:
+    from pip.operations.freeze import freeze as pip_freeze
 
-__version__ = '1.1.0'
+__version__ = '1.1.1'
```

Comment:
version is bumped in release commits only

Reply:
Will remember for the future, thx